belaUI.js

```diff
@@ -1258,6 +1258,7 @@ function notificationSendPersistent(conn) {
 
 /* Hardware monitoring */
 let sensors = {};
+const systemStartTime = Date.now();
```

Comment:
This will be incorrect if belaUI restarts, for example after an OTA update. I'll modify the code to parse `/proc/uptime` instead